pkg/controllers/dynakube/extension/consts/consts.go

```diff
@@ -20,11 +22,13 @@ const (
 	ExtensionsCollectorComPort        = 14599
 	ExtensionsCollectorTargetPortName = "collector-com"
 
-	ExtensionsCustomTlsCertificate = "custom-tls-certificates"
+	ExtensionsSelfSignedTlsSecretSuffix     = "-extensions-controller-tls"
+	ExtensionsSelfSignedTlsCommonNameSuffix = "-extensions-controller.dynatrace"
+	ExtensionsSelfSignedTlsRenewalThreshold = 12 * time.Hour
 
-	// TLSKeyDataName is the key used to store a TLS private key in the secret's data field.
-	TLSKeyDataName = "tls.key"
```

Comment:
any reason for this change?

Reply:
Just to keep all TLS references the same in code, as we have it like this from CRD `TlsRefName` for example
`dk.Spec.Templates.ExtensionExecutionController.TlsRefName`

Reply:
TLS is an abbreviation, so the correct camelCase is TLS. https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN

Reply:
it's a good point to make it similar, but if you try to grep current codebase you will find many inconsistency in different places:

```
er.p12`: certificate+key pair in pkcs12 format<br/>`password`: passphrase to read server.p12|-|string|

pkg/webhook/mutation/pod/oneagent/volumes_test.go
76:                                     TlsSecretName: "test",

assets/samples/dynakube/v1beta1/kubernetesObservability.yaml
140:    # Optional: Specifies the name of a secret containing a TLS certificate, a TLS key and the TLS key's password to be used by ActiveGate i
nstances
143:    # server.p12: TLS certificate and TLS key pair in pkcs12 format
144:    # password: passphrase to decrypt the TLS certificate and TLS key pair
146:    # tlsSecretName: "my-tls-secret"
```

Reply:
```
pkg/clients/dynatrace/client.go
5:      "crypto/tls"
163:// SkipCertificateValidation creates an Option that specifies whether validation of the server's TLS
169:                    if t.TLSClientConfig == nil {
170:                            t.TLSClientConfig = &tls.Config{} //nolint:gosec // fix is expected to be delivered soon
173:                    t.TLSClientConfig.InsecureSkipVerify = true
217:            if t.TLSClientConfig == nil {
218:                    t.TLSClientConfig = &tls.Config{} //nolint:gosec // fix is expected to be delivered soon
221:            t.TLSClientConfig.RootCAs = rootCAs

pkg/oci/registry/client_test.go
70:             assert.True(t, transport.TLSClientConfig.InsecureSkipVerify)
77:             assert.False(t, transport.TLSClientConfig.InsecureSkipVerify)

pkg/oci/registry/client.go
5:      "crypto/tls"
211:    if transport.TLSClientConfig == nil {
212:            transport.TLSClientConfig = &tls.Config{} //nolint:gosec
215:    transport.TLSClientConfig.RootCAs = rootCAs
221:    if transport.TLSClientConfig == nil {
222:            transport.TLSClientConfig = &tls.Config{} //nolint:gosec
225:    transport.TLSClientConfig.InsecureSkipVerify = skipCertCheck
```

Reply:
Its okay! I'm undoing it and making all TLS :+1:

Reply:
[Fixed](https://github.com/Dynatrace/dynatrace-operator/pull/3778/commits/f78d0db05eff5ec59242a155f958d6917857c392)

Reply:
thanks!